src/action_main.cpp

```diff
@@ -35,9 +35,17 @@ using namespace std::chrono_literals;
 int main(int argc, char * argv[])
 {
   rclcpp::init(argc, argv);
+  
+  const char * help_message = 
+    "Usage: ros2 run akushon action <path> <action_name>\n"
+    "  <path>        Path to the action configuration file\n"
+    "  <action_name> Name of the action to run\n";
 
-  if (argc < 2) {
-    std::cerr << "Please specify the path!" << std::endl;
+  std::string action_name = "";
```

Comment:
The `action_name` is not being modified after declaration, i guess it should extract a value from the arguments.